include/ppx/tri_mesh.h

```diff
@@ -60,6 +60,19 @@ struct TriMeshVertexData
     float3 bitangent;
 };
 
+//! @struct TriMesh16VertexData16
+//!
+//!
+struct TriMeshVertexData16
```

Comment:
Probably better with the name `TriMeshVertexDataCompressed`

Reply:
Done.